src/Utility/ConstraintStabiliser.php

```diff
@@ -95,12 +125,11 @@ public static function destabiliseConstraints(
         // Rewrite all dependencies.
         // Note: only rewrite dependencies for anything that was included in the release.
         // This mirrors functionality in PublishRelease::stabiliseConstraints()
-        $items = $releasePlanNode->getAllItems();
-        foreach ($items as $item) {
+        foreach (self::getAllItems($releasePlanNode) as $item) {
```

Comment:
As above, use `$libraryRelease->getRootRelease()->getAllItems();` to get all the items

Reply:
Updated